src/cloud/internal/abstractcloudservice.h

```diff
@@ -67,15 +67,51 @@ class CloudService : public QObject, public IAuthorizationService, public ICloud
 
     Ret checkCloudIsAvailable() const override;
 
-    framework::ProgressPtr uploadScore(QIODevice& scoreData, const QString& title, Visibility visibility = Visibility::Private,
-                                       const QUrl& sourceUrl = QUrl()) override;
-    framework::ProgressPtr uploadAudio(QIODevice& audioData, const QString& audioFormat, const QUrl& sourceUrl) override;
-
-    RetVal<ScoreInfo> downloadScoreInfo(const QUrl& sourceUrl) override;
-
 private slots:
     void onUserAuthorized();
 
+protected:
+    struct ServerConfig {
```

Comment:
Same issue: https://github.com/musescore/MuseScore/pull/17346#discussion_r1194749156

It's not a configuration